src/middlewared/middlewared/plugins/failover_/zpool_cachefile.py

```diff
@@ -18,8 +17,7 @@ class Config:
         private = True
         namespace = 'failover.zpool.cachefile'
 
-    @accepts(Str('event', enum=['MASTER', 'BACKUP', 'SYNC'], default='MASTER'))
-    def setup(self, event):
+    def setup(self, event: str = "MASTER"):
```

Comment:
Could we do `Literal["MASTER", "BACKUP", "SYNC"]`?

Reply:
Sure, done!